addons/l10n_mz/data/tax_report.xml

```diff
@@ -0,0 +1,217 @@
+                        </field>
+                    </record>
+                    <record id="l10n_mz_tax_report_line_sum" model="account.report.line">
+                        <field name="name">Settlement Amount</field>
+                        <field name="children_ids">
+                            <record id="l10n_mz_tax_report_field_11" model="account.report.line">
+                                <field name="name">Field 11 - Tax base Total</field>
+                                <field name="expression_ids">
+                                    <record id="l10n_mz_tax_report_11_tag" model="account.report.expression">
+                                        <field name="label">balance</field>
+                                        <field name="engine">aggregation</field>
+                                        <field name="formula">MZ_TR_F_1.balance + MZ_TR_F_3.balance + MZ_TR_F_4.balance</field>
+                                    </record>
+                                </field>
+                            </record>
+                            <record id="l10n_mz_tax_report_field_12" model="account.report.line">
+                                <field name="name">Field 12 - Total tax in favor of the taxable person</field>
+                                <field name="code">MZ_TR_F_12</field>
+                                <field name="expression_ids">
+                                    <record id="l10n_mz_tax_report_12_tag" model="account.report.expression">
+                                        <field name="label">balance</field>
+                                        <field name="engine">aggregation</field>
+                                        <field name="formula">MZ_TR_F_5.balance + MZ_TR_F_6.balance + MZ_TR_F_7.balance + MZ_TR_F_8.balance + MZ_TR_F_9.balance</field>
+                                    </record>
+                                </field>
+                            </record>
+                            <record id="l10n_mz_tax_report_field_13" model="account.report.line">
+                                <field name="name">Field 13 - Total tax in favor of the state</field>
+                                <field name="code">MZ_TR_F_13</field>
+                                <field name="expression_ids">
+                                    <record id="l10n_mz_tax_report_13_tag" model="account.report.expression">
+                                        <field name="label">balance</field>
+                                        <field name="engine">aggregation</field>
+                                        <field name="formula">MZ_TR_F_2.balance + MZ_TR_F_10.balance</field>
+                                    </record>
+                                </field>
+                            </record>
+                        </field>
+                    </record>
+                    <record id="l10n_mz_tax_report_line_final" model="account.report.line">
```

Comment:
Why not `l10n_mz_tax_report_field_14` ?

Reply:
nah final contains feild 14 and feild 15